index.html

```diff
@@ -82,13 +82,7 @@ <h5 class="light grey-text text-lighten-3"></h5>
       </ul>
     </div>
   </section>
-  <footer class="page-footer #b71c1c red darken-4">
-    <div class="footer-copyright ">
-      <div class="container">
-        © 2020 Copyright
-      </div>
-    </div>
-  </footer>
```

Comment:
just change this to:

`<footer class="page-footer #b71c1c red darken-4">
            <div class="container">
              © 2020 Copyright 
            </div>
          </footer>`